src/Entity.cs

```diff
@@ -15,6 +15,7 @@ public class Entity
             {"payment.downtime", "Razorpay.Api.Payment" },
             {"refund", "Razorpay.Api.Refund"},
             {"order", "Razorpay.Api.Order"},
+            {"order.fulfillment", "Razorpay.Api.Order"},

```

Comment:
Didn't get this. How is this needed?

Reply:
according to this response 
https://razorpay.com/docs/payments/magic-checkout/rto-intelligence/#step-3-update-the-fulfillment-details
```
{
  "entity": "order.fulfillment",
  "order_id": "EKwxwAgItXXXX",
  "payment_method": "upi",
  "shipping": {
    "waybill": "123456789",
    "status": "rto",
    "provider": "Bluedart"
  }
}
```